cmd/util/ledger/reporters/fungible_token_tracker.go

```diff
@@ -192,8 +195,9 @@ func (r *FungibleTokenTracker) iterateChildren(tr trace, addr flow.Address, valu
 		}
 
 		// iterate over fields of the composite value (skip the ones that are not resource typed)
-		compValue.ForEachField(func(key string, value interpreter.Value) {
-			r.iterateChildren(append(tr, key), addr, value)
-		})
+		compValue.ForEachField(&interpreter.Interpreter{},
```

Comment:
Why make a new interpreter when there's one above, as `inter`?

Reply:
Going to put the fix for this into Secure Cadence - integration test fix #2338

Reply:
641687c505f22065e3b3b7f10cb4c9bd21fdaa7f